server/util/capabilities/capabilities.go

```diff
@@ -0,0 +1,26 @@
+package capabilities
+
+import (
+	akpb "github.com/buildbuddy-io/buildbuddy/proto/api_key"
+)
+
+func FromInt(m int32) []akpb.ApiKey_Capability {
+	caps := []akpb.ApiKey_Capability{}
+	c := int32(1)
+	for m > 0 {
+		if m&1 > 0 {
+			caps = append(caps, akpb.ApiKey_Capability(c))
+		}
+		m >>= 1
+		c <<= 1
+	}
```

Comment:
had to convince myself that this works but it does :)

another way is to say:

for _, cap := range allCaps {
  if cap & m > 0 {
    caps = append(caps, cap)
  }
return caps

Reply:
Ah yeah that looks safer; not as much casting. And shorter too! Done